### DISCUSSION TITLE
Using Extension on Non-Default Database

### DISCUSSION THREAD

**Discussion by lewysigns** [firebase/extensions#2160] [Q&A]
Is there a way to have the extension listen to a database that is NOT the default database? When I add it to my project it will only map to the default database where I do not want it.

I'm trying to use the translate extension, which fires a when a new document is added to the translation collection, but only fires if that collection is in the default database.

Thanks!

**Comment by gmyrianthous** [comment]
Hey, have you found any workaround by any chance?

**Reply by lewysigns** [reply]
There is a way, but it has to be a parameter built into the extension.

This one did not, instead I used the Google translate api server side and stored the result, listening for that specific document.

**Reply by cabljac** [reply]
Hi there,

This behaviour is only supported in v2 firebase functions I believe, which in turn aren't supported for extensions yet.
